### DISCUSSION TITLE
PINT support

### DISCUSSION THREAD

**Discussion by AlexandruCiobanu** [phax/phoss-smp#274] [General]
Hello Phil,
I would like to know if the SMP supports PINT. Also on the sender side, does the as4 peppol sender support looking up identifiers in the order from the most specialized to the least specialized id or is that something than one would need to implement ?

Thank you,
Alex

**Comment by phax** [comment]
Hi Alex,
Yes inded, phoss SMP supports the registration of `peppol-doctype-wildcard` identifiers including the `*` in the document type identifier values. Please note, that the consistency checks for that are not yet perfect, so you need to do it right.

And yes, the Peppol `SMPClient` class supports via `getWildcardServiceMetadataOrNull` the resolution of wildcard identifiers.
